ts/features/wallet/component/features/PagoPaPaymentCapability.tsx

```diff
@@ -126,26 +124,15 @@ const PagoPaPaymentCapability: React.FC<Props> = props => {
   );
 
   return (
-    <TouchableDefaultOpacity onPress={present}>
-      <View style={styles.row}>
-        <View style={styles.left}>
-          <H4
-            weight={"SemiBold"}
-            color={"bluegreyDark"}
-            testID={"PagoPaPaymentCapabilityTitle"}
-          >
-            {I18n.t("wallet.methods.card.pagoPaCapability.title")}
-          </H4>
-          <H5
-            weight={"Regular"}
-            color={"bluegrey"}
-            testID={"PagoPaPaymentCapabilityDescription"}
-          >
-            {I18n.t("wallet.methods.card.pagoPaCapability.description")}
-          </H5>
-        </View>
-        {availabilityBadge(isPaymentMethodSupported(props.paymentMethod))}
-      </View>
+    <TouchableDefaultOpacity
+      onPress={paymentSupported === "available" ? undefined : present}
```

Comment:
```suggestion
      onPress={(paymentSupported === "available" && pmActivatePaymentEnabled) ? undefined : present}
```
If I well understood

Reply:
You're right, changed in 9ede69a32d13b166d7596860c13dd115de5fa66d